gatsby-ssr.tsx

```diff
@@ -53,7 +53,7 @@ function SentryLoaderConfig() {
       key="sentry-loader-config"
       dangerouslySetInnerHTML={{
         __html: `
-Sentry.onLoad(function() {
+Sentry && Sentry.onLoad(function() {
```

Comment:
This will throw if `Sentry` is not defined in the global scope yet. I think we either need to do this:

```suggestion
window.Sentry && Sentry.onLoad(function() {
```

or this:

```suggestion
typeof Sentry !== 'undefined' && Sentry.onLoad(function() {
```

Reply:
good point! I'll update it to `window.Sentry` (a bit less verbose)